### PR TITLE
Translate content/pt/docs/concepts/cluster-administration/logging.md into Portugues 

### PR DESCRIPTION
**Pull request by jhonmike**
- Translated documentation referent logs
- Copied examples files used in the doc

### CODE REVIEW

File: content/pt/docs/concepts/cluster-administration/logging.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+As arquiteturas de log no nível de cluster são descritas no pressuposto de que um back-end de log esteja presente dentro ou fora do cluster. Se você não estiver interessado em ter o log no nível do cluster, ainda poderá encontrar a descrição de como os logs são armazenados e manipulados no nó para serem úteis.
+Cluster-level logging architectures are described in assumption that a logging backend is present inside or outside of your cluster. If you're not interested in having cluster-level logging, you might still find the description of how logs are stored and handled on the node to be useful.
```
Comment (femrtnz) [inline content/pt/docs/concepts/cluster-administration/logging.md]: I think you missed this in english :)

File: content/pt/docs/concepts/cluster-administration/logging.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+## Log básico no Kubernentes
+
+Nesta seção, você pode ver um exemplo de log básico no Kubernetes que gera dados para o fluxo de saída padrão. Esta demostração usa uma [especificação de pod](/examples/debug/counter-pod.yaml) com um contêiner que grava algum texto na saída padrão uma vez por segundo.
```
Comment (femrtnz) [inline content/pt/docs/concepts/cluster-administration/logging.md]: What if we include thise as well`... fluxo de saída padrão (standard output stream)...` ?

File: content/pt/docs/concepts/cluster-administration/logging.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+A saída será:
+
+```
+0: Mon Jan  1 00:00:00 UTC 2001
+1: Mon Jan  1 00:00:01 UTC 2001
+2: Mon Jan  1 00:00:02 UTC 2001
+...
+```
+
+Você pode usar `kubectl logs` para recuperar logs de uma instanciação anterior de um contêiner com o sinalizador `--previous`, caso o contêiner tenha falhado. Se o seu pod tiver vários contêineres, você deverá especificar quais logs do contêiner você deverá especificar quais logs do contêiner você deseja acessar anexando um nome de contêiner ao comando. Veja a [documentação do `kubectl logs`](/docs/reference/generated/kubectl/kubectl-commands#logs) para mais destalhes.
```
Comment (femrtnz) [inline content/pt/docs/concepts/cluster-administration/logging.md]: I believe this is a bit confusing

`... você deverá especificar quais logs do contêiner você deverá especificar quais logs do contêiner você deseja acessar anexando um nome de contêiner ao comando.`

File: content/pt/docs/concepts/cluster-administration/logging.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+Por exemplo, nos clusters do Kubernetes, implementados pelo script `kube-up.sh`, existe uma ferramenta [`logrotate`](https://linux.die.net/man/8/logrotate) configurada para executar a cada hora. Você pode configurar um tempo de execução do contêiner para girar os logs do aplicativo automaticamente, por exemplo, usando o `log-opt` do Docker.
+No script `kube-up.sh`, a última abordagem é usada para imagem COS no GCP, e a anterior é usada em qualquer outro ambiente. Nos dois casos por padrão, a rotação é configurada para ocorrer quando o arquivo de log exceder 10MB.
+
+Como exemplo, você pode encontrar informações detalhadas sobre como o `kube-up.sh` define o log da imagem COS no GCP no [script][cosconfigurehelper] correspondente.
+
+Quando você executa [`kubectl logs`](/docs/reference/generated/kubectl/kubectl-commands#logs) como em exemplo básico de log, o kubelet no nó lida com a solicitação e lê diretamente do arquivo de log, retornando o conteúdo na resposta.
```
Comment (femrtnz) [inline content/pt/docs/concepts/cluster-administration/logging.md]: What if we change ` como em exemplo básico de log` to ` como no exemplo de log basico acima`

File: content/pt/docs/concepts/cluster-administration/logging.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+Atualmente, se algum sistema externo executou a rotação, apenas o conteúdo do arquivo de log mais recente estará disponível através de `kubectl logs`. Por exemplo, se houver um arquivo de 10MB, o `logrotate` executa a rotação e existem dois arquivos, um com 10MB de tamanho e um vazio, o `kubectl logs` retornará uma resposta vazia.
+{{< /note >}}
+
+[cosConfigureHelper]: https://github.com/kubernetes/kubernetes/blob/{{< param "githubbranch" >}}/cluster/gce/gci/configure-helper.sh
+
+### Logs de componentes do sistema
+
+Existem dois tipos de componentes do sistema: aqueles que são executados em um contêiner e aqueles que não são executados em um contêiner. Por exemplo:
+
+- O agendador Kubernetes e o kube-proxy são executados em um contêiner.
```
Comment (femrtnz) [inline content/pt/docs/concepts/cluster-administration/logging.md]: I'm not sure we should translate the scheduler or not, as this is the component name. Any thoughts?

File: content/pt/docs/concepts/cluster-administration/logging.md (hunk 2 above)
Comment (jhonmike) [reply content/pt/docs/concepts/cluster-administration/logging.md]: LGTM

File: content/pt/docs/concepts/cluster-administration/logging.md (hunk 3 above)
Comment (jhonmike) [reply content/pt/docs/concepts/cluster-administration/logging.md]: I believe that during the translation, I doubled the thought hehe

File: content/pt/docs/concepts/cluster-administration/logging.md (hunk 5 above)
Comment (jhonmike) [reply content/pt/docs/concepts/cluster-administration/logging.md]: I think it is better not to translate, the same way I did with service, pod, deployment... I'll adjust!

**Comment by devlware** [comment]
I'll review the PR later today.

**Pull request merged**